crates/router/src/core/admin.rs

```diff
@@ -3501,6 +3501,8 @@ impl BusinessProfileCreateBridge for api::BusinessProfileCreate {
                 .or(Some(false)),
             outgoing_webhook_custom_http_headers: outgoing_webhook_custom_http_headers
                 .map(Into::into),
+            tax_connector_id: None,
```

Comment:
Shouldn't the `tax_connector_id` field be populated from the request, since you accept it in the `BusinessProfileCreate` API model?